hack/verify-licenses.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: `hack/verify-licenses.sh`.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+
+kube::golang::verify_go_version
+kube::util::ensure-temp-dir
+
+
+# Creating a new repository tree 
+# Deleting vendor directory to make go-licenses fetch license URLs from go-packages source repository
+git worktree add -f "${KUBE_TEMP}"/tmp_test_licenses/kubernetes HEAD >/dev/null 2>&1 || true
+cd "${KUBE_TEMP}"/tmp_test_licenses/kubernetes && rm -rf vendor
+
+
+# Ensure that we find the binaries we build before anything else.
+export GOBIN="${KUBE_OUTPUT_BINPATH}"
+PATH="${GOBIN}:${PATH}"
+
+
+# Explicitly opt into go modules, even though we're inside a GOPATH directory
+export GO111MODULE=on
+
+
+allowed_licenses=()
+packages_flagged=()
+packages_url_missing=()
+exit_code=0
+
+git remote add licenses https://github.com/kubernetes/kubernetes >/dev/null 2>&1 || true
+
+
+# Install go-licenses
+echo -e '[INFO] Installing go-licenses...'
```

Comment:
```suggestion
echo '[INFO] Installing go-licenses...'
```

Do you really need the `-e` here?

Reply:
Removed all instances where -e not required. Thanks for pointing.